src/_includes/layouts/topic-page.njk

```diff
@@ -11,18 +11,18 @@
 {% endblock %}
 
 {% block content %}
-    {%- article_contents content, translations[lang].articleContentsSummary -%}
+    {%- article_contents locale, content -%}
```

Comment:
I would revert the changes to this shortcode and pass `_('Article Contents')` as the second parameter rather than localizing the string within the shortcode.

Reply:
Thanks @greatislander, that makes sense. I updated but I was getting a lint error about '_' is undefiend, so I used eslint-disable-next-line.

Reply:
```suggestion
    {%- article_contents content, _('Article Contents') -%}
```